lib/numbers.md

```diff
@@ -5,20 +5,25 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+In Ruby, there are integers (“fixnum”) which are whole numbers, and there are floats which have decimal points.
+
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+You can do all the familiar math operations on integers/floats in Ruby (add, subtract, multiply, divide, etc.). You can also use methods to do things like convert floats to integers and vice versa.
+
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+If you add (+) two strings together, you’ll basically get the two strings mashed together. So if they’re words, say, “cat” and “dog,” adding those two strings will give you “catdog.” If your strings are digits like “4” and “5,” adding them will give you “45.” However, if they are numbers and not strings, traditional math rules apply (4 + 5 = 9, not 45). 
+
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+Sometimes it will work. You can multiply a string. If you took your string “20” and multiplied it by four (“20” * 4) you would get “20202020.” But you couldn’t multiply the number 4 by your string “20” (4 * “20”). That will give you an error since “20” as a string is the same as any other group of characters acting as a string and you couldn’t get an answer for 4 multiplied by the string “cat,” for example. Division won’t work, either. As far as I can tell, you can only add and multiply strings. There’s always the option to convert string digits to integers (.to_i) and perform mathematical operations on them, though. 
+
```

Comment:
What if you had a string `"20"` and wanted to divide it by **3**? What problem would using `to_i` cause? How would you avoid that problem?